firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -17,11 +17,15 @@
 import android.text.TextUtils;
 import com.google.firebase.installations.local.PersistedInstallationEntry;
 import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
```

Comment:
Please check with Android folks (e.g. Ashwin) if we can use that package here of if we should use re2j.

Reply:
Greg said it's used in FCM SDK so it's fine.

Reply:
He might have said that because it's GMSCore and GMSCore has rules that apply to older code.
Maybe for other Firebase SDKs we have other requirements.
Please check with someone from Android team, if it is fine to use this package instead of `re2j`.